### PR TITLE
[Kibana] remove useless maxUnavailable in Kibana chart

### PR DESCRIPTION
**Pull request by victorsalaun**
This PR removes useless `maxUnavailable` attribute in Kibana chart
Solves #331 

- [X] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [X] README.md updated with any new values or changes
- [X] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins test this please

File: kibana/tests/kibana_test.py [hunk 1]
```diff
@@ -191,15 +191,11 @@ def test_override_the_default_update_strategy():
     config = '''
 updateStrategy:
   type: "RollingUpdate"
-  rollingUpdate:
```
[the remark this answers is not in the document]
Comment (victorsalaun) [reply kibana/tests/kibana_test.py]: Nice catch, indeed same name `maxUnavailable` but for different usage.
I reverted the deletion of the test.

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by victorsalaun** [comment]
Thanks @jmlrt

**Pull request merged**